dnase/trackhub/update_tracks.bash

```diff
@@ -294,12 +302,19 @@ else
     blatport=0
 fi
 
-if [ ${blatport} -ne 0 ]; then
+if [ "${blatport}" != "0" ]; then
     echo "Copying ${customGenomeAssembly}.2bit to cadlej01_shared..."
     cp -p ${hub_target_final}/${customGenomeAssembly}/data/${customGenomeAssembly}.2bit /vol/isg/cadlej01_shared
-    
+    twoBitFiles="${customGenomeAssembly}.2bit"
+
+    if [ "${blatport}" = "17779" ]; then
```

Comment:
what's this testing?

Reply:
17779 is the port number assigned to the blat server for the CEGS hub.  If we are building a hub for that port, we want to add t2t as well.

Reply:
what other blat servers are there?

Reply:
17778 for MAURANOLAB.

Reply:
don't key off of secondary features. check this way instead:
if [[ "${hub_type}" == "CEGS" ]]; then